drone/oc-integration-tests/storage-oc.toml

```diff
@@ -23,6 +23,7 @@ data_server_url = "http://localhost:11001/data"
 [grpc.services.storageprovider.drivers.owncloud]
 datadirectory = "/drone/src/tmp/reva/data"
 redis = "redis:6379"
+userprovidersvc = "localhost:18000"
```

Comment:
Setting this will save a trip to the gateway. When left out it should fall back to the `gatewaysvc` in the `[shared]` section and work just as well.

Reply:
the fallback did not seem to work, I had to add this

there was a strange thing where it was resolving some service, but that service did not implement the UserAPI, see https://github.com/cs3org/reva/pull/1033#issuecomment-669823201

I didn't manage to find out what service it was resolving exactly